stories/page-footer/page-footer.stories.js

```diff
@@ -0,0 +1,215 @@
+			<a class="fd-link fd-page-footer__row--item" id="fd-link-5" href="/" title="Participant Team">Participant Team</a>
+        </div>
+		<div class="fd-page-footer__text">
+			<span class="fd-form-label"> 1992-2021 @copyright all right reserved.</span>
+		</div>
+	</div>
+</div>
+`;
+
+singleLineLink.parameters = {
+    docs: {
+        iframeHeight: 100,
+        storyDescription:
+            'The Page Footer contains Image/logo, And list of links in the footer. It is responsive to tablet and mobile screen sizes.'
+    }
+};
+
+export const MultiLineLink = () => `
+<div class="fd-page-footer fd-page-footer--xl">
+	<div class="fd-page-footer__logo">	
+		<img src="//unpkg.com/fundamental-styles/dist/images/sap-logo.png" alt="page footer logo"></img>
+	</div>
+	<div class="fd-page-footer__container">
+		<div class="fd-page-footer__row">
+			<a class="fd-page-footer__row--item fd-link" id="fd-link-6" href="/" title="Supported Browser">Supported Browser</a>
+			<a class="fd-page-footer__row--item fd-link" id="fd-link-7" href="/" title="Security Disclosure">Security Disclosure</a>
+		</div>
+		<div class="fd-page-footer__row">
+			<a class="fd-page-footer__row--item fd-link" id="fd-link-8" href="/" title="Supported Browser">Supported Browser</a>
+			<a class="fd-page-footer__row--item fd-link" id="fd-link-9" href="/" title="Security Disclosure">Security Disclosure</a>
+			<a class="fd-page-footer__row--item fd-link" id="fd-link-10" href="/" title="Privacy Statement">Privacy Statement</a>
+			<a class="fd-page-footer__row--item fd-link" id="fd-link-11" href="/" title="Cookie Statement">Cookie Statement</a>
+			<a class="fd-page-footer__row--item fd-link" id="fd-link-12" href="/" title="Participant Team">Participant Team</a>
+		</div>
+		<div class="fd-page-footer__text">
+			<span class="fd-form-label"> 1992-2021 @copyright all right reserved.</span>
+		</div>
+	</div>
+</div>
+`;
+
+MultiLineLink.parameters = {
+    docs: {
+        iframeHeight: 100,
+        storyDescription:
+            'The Page Footer contains Image/logo, And list of links in the footer. To apply multiline link, add the <code class="docs-code">fd-page-footer__row</code> modifier class to the container element.'
+    }
+};
+
+export const ImageAndIconLink = () => `
+<div class="fd-page-footer fd-page-footer--xl">
+	<div class="fd-page-footer__logo">	
+		<img src="//unpkg.com/fundamental-styles/dist/images/sap-logo.png" alt="page footer logo"></img>
+	</div>
+	<div class="fd-page-footer__container">
+		<div class="fd-page-footer__row">
+			<a href="#" id="fd-link-13"  class="fd-page-footer__row--item fd-link" tabindex="0">
+			<span class="sap-icon--download sap-icon--s"></span>
+			 icon and link
+			</a>
+		</div>
+		<div class="fd-page-footer__row">
+			<a href="#" id="fd-link-14" class="fd-page-footer__row--item fd-page-footer__row--item--image fd-link" tabindex="0">
```

Comment:
Please check the BEM naming!
`fd-page-footer__row--item--image` is not BEM. You can't have 2 modifiers.
`fd-page-footer` is your BLOCK
`fd-page-footer__row` is your ELEMENT, that's why it has `__`
`fd-page-footer__row--item` is your modifier, that's why you have `--`

But you can't add a second modifier or a second element. 
You need to change `fd-page-footer__row--item` to `fd-page-footer__row-item`, in this case `__row-item` is your element and then have `fd-page-footer__row-item--image` as a modifier.